### PR TITLE
Promote local to parameter

### PR DESCRIPTION
**Pull request by Hosch250**

### CODE REVIEW

File: RetailCoder.VBE/App.cs [hunk 1]
```diff
@@ -56,36 +57,104 @@ public class App : IDisposable
...
+                if (_skipKeyUp)
+                {
+                    _skipKeyUp = false;
+                    return;
+                }
+
+                if (_isAwaitingTwoStepKey)
+                {
+                    // todo: use _firstStepHotKey and e.Key to run 2-step hotkey action
+                    if (_firstStepHotKey == Keys.I && e.Key == Keys.M)
```
Comment (Vogel612) [inline RetailCoder.VBE/App.cs]: I'd even go so far to try and get Keystrokes (or multi-step-keys) into a configuration based mapping, something like a properly encapsuled `Dictionary<Keystroke, Command>`. Hardcoding this doesn't seem like a viable option to me

Maybe that's only for a next step, though

Comment (Hosch250) [inline RetailCoder.VBE/App.cs]: That isn't mine.  It is from @retailcoder's Smart Indenter branch and is thoroughly broken (that is why it is commented out lower down and never initialized.

Comment (retailcoder) [inline RetailCoder.VBE/App.cs]: *cough* for some values of "thoroughly broken" :wink:

@Vogel612 that's precisely the plan :smile: ...I happened to bump into some difficulties putting it all together. Hotkeys are correctly captured and update the "status bar", but the Ctrl/Shift modifiers "stick" and the 2nd key becomes impossible to capture, not to mention how annoying it becomes to have Ctrl/Shift "stuck". As for the low-level keyboard hook, it works but tends to destabilize the whole environment and has been identified as a source of unexpected crashes.

Comment (Hosch250) [inline RetailCoder.VBE/App.cs]: For some values of "unexpected crashes".  It crashes Excel every time I open the VBE with that code uncommented.

**Comment by retailcoder** [comment]
I'll merge this one later, apparently there are conflicts.. cannot merge from phone.. @ckuhn203 @Hosch250 feel free to merge it

**Pull request merged**